### DISCUSSION TITLE
songs Meta Data - do I have to update properties>Details with Aritists name, Album, Album Years

### DISCUSSION THREAD

**Discussion by hercules108** [ampache/ampache#3758] [Q&A] [answered]
Can you please elaborate on mp3 song when you upload to Storage Volume? Do I  have to update each mp3 song's properties>Details with Aritists name, Album, Album Years? How does code work on displaying Artists lists and their related songs? Do I have to update each songs Meta Data by going in their Properties as shown in following picture?
![image](https://github.com/ampache/ampache/assets/155604865/07b36b2f-83f9-49a0-9d77-121b6c113da1)

**Comment by mitchray** [comment, marked as answer]
You would want to have all that information filled yes, you can use something like [MusicBrainz Picard](https://picard.musicbrainz.org/) to analyze and fill this for you
